skema/program_analysis/CAST2FN/ann_cast/to_gromet_pass.py

```diff
@@ -3709,9 +3729,9 @@ def visit_model_import(
             else:
                 self.import_collection[name] = (alias, [symbol], all)
                 # We also maintain the symbol as a 'variable' of sorts in the global environment
-                # self.add_import_symbol_to_env(
-                #    symbol, parent_gromet_fn, parent_cast_node
-                #)
+                self.add_import_symbol_to_env(
```

Comment:
this seems to mirror the same the if condition.  Can we instead follow the if / else with 

```python
self.add_import_symbol_to_env(
    symbol, parent_gromet_fn, parent_cast_node
)
```